upgrade/sql/1.7.0.0.sql

```diff
@@ -154,7 +154,7 @@ CREATE TABLE `PREFIX_module_access` (
 
 /* PHP:add_quick_access_tab(); */;
 
-INSERT INTO `PREFIX_hook` (`name`, `title`, `description`, `position`) VALUES
+INSERT IGNORE INTO `PREFIX_hook` (`name`, `title`, `description`, `position`) VALUES
```

Comment:
What about REPLACE to force changes from the upgrade, instead of keeping the previous obsolete values ?

Reply:
The important thing is the name column, I don't thing we need to write a replace query for this?

Reply:
Not 100% sure either replacing is important. I just assumed that upgrade could fix a typo in description, or update the position.

Reply:
I think the position column was related to if the hook allowed drag and drop of the order of the modules.

Reply:
Most likely yes, then the command shouldn't replace current position. Your PR is fine :)